Adding Kong's contribution swag

preethamvishy opened the pull request:
- [x] I've checked that this isn't a duplicate pull request.

Is it regarding a new swag proposal or something else?  
Fill the corresponding section below:

(New Swag Proposal)

* Company: Kong
* Reference URL: https://discuss.konghq.com/t/contribute-to-kong-for-hacktoberfest/2029

Q. Have you received swag from this?  No
Q. How long did it take?  Unknown
Q. Does it ship worldwide?  Unknown

(Something else)  
Please describe your changes here:

vikaspotluri123 commented on data.json:
```diff
@@ -55,6 +55,14 @@
+    "difficulty": "medium",
+    "description": "Everyone who gets 1 PR accepted to a Kong repo can get a sweet Kong Contributor T-shirt!",
+    "reference": "https://discuss.konghq.com/t/contribute-to-kong-for-hacktoberfest/2029",
+    "image": "https://user-images.githubusercontent.com/121322/45907730-f6a80b00-bdad-11e8-93ef-774392192716.png",
```
I don't think using the hacktoberfest logo is suitable because there's an implication that's how the shirt looks.

I did some digging and it looks like they're offering this shirt:

https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt

If someone could double verify that would be awesome 👌

preethamvishy replied:
Thanks for digging into this!
Kong has a separate [blog post for Hacktoberfest](https://discuss.konghq.com/t/contribute-to-kong-for-hacktoberfest/2029) mentioning "a sweet Kong Contributor T-shirt" and this one seems to be from the general contribution guidelines. Could these be different?

Open to modifying the PR or adding a separate PR depending on what the community feels.

aslafy-z replied:
You can modify this PR directly :wink:

preethamvishy replied:
I meant:
I can add a separate PR if the community feels that Kong's Hacktoberfest and general contribution t-shirt giveaways are independent.
Or, if the community feels that both essentially the same or related, I will modify this PR itself. 

Apologies for the ambiguity earlier.

preethamvishy replied:
Modified this PR anyway.

preethamvishy commented:
Kong's contribution swag seems to be independent of Hacktoberfest. Removed Hacktoberfest and added their general contribution reference and image, thanks to @vikaspotluri123.

aslafy-z commented on data.json:
```diff
@@ -55,6 +55,14 @@
+    "difficulty": "medium",
+    "description": "Everyone who gets 1 PR accepted to a Kong repo can get a Kong Contributor T-shirt!",
+    "reference": "https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt",
+    "image": "https://camo.githubusercontent.com/2390aa650f4b56a417db70025153adabd6234cf3/68747470733a2f2f6b6f6e6768712e636f6d2f77702d636f6e74656e742f75706c6f6164732f323031382f30342f3130302d636f6e7472696275746f722d742d73686972742d31303234783736382e6a7067",
```
Please get the real link from within the markdown source.

aslafy-z commented:
I think you could let the hadktoberfest tag as it could count towards their quotas and it looks like any other contributing event right ? thanks

aslafy-z commented:
We may also want to keep the original reference url  ? Why you changed it ?

preethamvishy commented:
The original reference is Hacktoberfest-specific. The current reference leads to their more inclusive contributor t-shirt program.

aslafy-z commented:
@preethamvishy Could you please fix conflicts & rebase? Thanks!
